Source/WebCore/dom/make_names.pl

```diff
@@ -1012,6 +1040,85 @@ sub printTagNameCppFile
     close F;
 }
 
+sub printAttributeNameHeaderFile
+{
+    my ($headerPath) = shift;
+    my $F;
+    open F, ">$headerPath";
+
+    printLicenseHeader($F);
+    print F "#pragma once\n";
+    print F "\n";
+    print F "#include \"Namespace.h\"\n";
```

Comment:
Can we use a forward declaration here instead?

Reply:
Probably, will try.